packages/components/src/Dialog/Layout/DialogLayout.tsx

```diff
@@ -25,61 +25,59 @@
  */
 
 import React, { FC, ReactNode } from 'react'
-import { Spinner } from '../../Spinner'
+import { ModalHeaderProps } from '../../Modal/ModalHeader'
+import {
+  ModalLayout,
+  ModalLayoutProps,
+  ModalLoading,
+} from '../../Modal/ModalLayout'
 import { DialogContent } from './DialogContent'
 import { DialogFooter } from './DialogFooter'
 import { DialogHeader, DialogHeaderProps } from './DialogHeader'
 
-export interface DialogLayoutProps {
-  /**
-   * Content to be displayed in footer
-   */
-  footer?: ReactNode
+export interface DialogLayoutProps extends ModalLayoutProps {
   /**
    * Secondary content to place in the footer
    * NOTE: `footer` property must be supplied for footer to be displayed. Supplying
    * only `footerSecondary` will prevent `DialogFooter` from being displayed.
    */
   footerSecondary?: ReactNode
 
-  /**
+  /*
```

Comment:
Annoying detail but to be recognized as a JSDoc (used by Storybook and IDEs to show documentation on prop) a comment needs two leading asterix

```suggestion
  /**
```